_episodes/15-practices.md

```diff
@@ -22,7 +22,47 @@ no different from drama, music, or athletics.
 And as in those fields,
 we have a collection of small tips and tricks to make teaching work better.
 
-### Challenges
+## The Code of Conduct
+
+Software Carpentry and Data Carpentry have a shared code of conduct
+(online [here]({{ site.swc_site }}/conduct/) and [here]({{ site.dc_site }}/code-of-conduct/))
+and participants in our workshops are required to abide by it.
+Hosts *must* point people at it during registration,
+and instructors *must* remind attendees of it at the start of the workshop.
+The Code of Conduct doesn't just tell everyone what the rules are:
+it tells them that there *are* rules,
+and that they can therefore expect a comfortable learning experience.
```

Comment:
I think "comfortable" pertains to the learning environment, not to the learning (experience) itself -- so how about "... they can therefore expect to learn in a comfortable (safe / welcoming / harassment free) environment"?